src/data/default.json

```diff
@@ -329,7 +329,8 @@
       "10": { "name": "local", "icon": "pi pi-cog" },
       "20": { "name": "online", "icon": "pi pi-globe" }
     },
-    "servicesStarted": false
+    "servicesStarted": false,
+    "serviceAvailable": false
```

Comment:
what does this variable refers to ?

Reply:
This is variable to set boolean  once the typeDB init and typeDB sample is in available status

Reply:
Done as discussed